content/MANUAL.md

```diff
@@ -1,61 +1,316 @@
-# Manual for the template
+# Project Manual
 
-If you are looking for documentation about the project, you should refer to the [README.md](./README.md) file.
+This manual provides detailed information about using and maintaining your Fable binding project. For a quick overview, refer to the [README.md](./README.md) file.
 
-## Architecture
+## Table of Contents
 
-- `src/` contains the source code of your binding
-- `demo/` contains a demo application that you can use to test your binding. It can also be publish on GitHub pages to have an interactive demo.
-- `build/` contains the build process for this repository.
+- [Project Structure](#project-structure)
+- [Configuration Files](#configuration-files)
+- [Development Workflow](#development-workflow)
+- [Building and Testing](#building-and-testing)
+- [Deployment and Publishing](#deployment-and-publishing)
+- [Troubleshooting](#troubleshooting)
 
-    It allows you to:
+## Project Structure
 
-    - Run the demo locally
-    - Build and publish the demo on GitHub pages
-    - Make a release of your binding on NuGet (it will also publish the demo on GitHub pages)
+```
+YourProject/
+├── src/                # Source code of your binding
+│   ├── YourProject.fs  # Main binding code
+│   └── YourProject.fsproj  # Project file for the binding
+├── demo/               # Demo application
+│   ├── Main.fs         # Demo application entry point
+│   ├── index.html      # Demo HTML template
+│   └── vite.config.ts  # Vite configuration
+├── build/              # Build scripts
+│   ├── Commands/       # Build commands implementation
+│   └── EasyBuild.fsproj  # Build project
+├── Directory.*.props   # MSBuild property files
+├── build.cmd           # Windows build script
+└── build.sh            # Unix build script
+```
+
+### Key Directories
+
+- **src/**: Contains all the source code for your binding. This is where you'll spend most of your development time.
+  
+- **demo/**: Contains a demo application that showcases your binding in action. The demo can be:
+  - Run locally during development
+  - Published to GitHub Pages as an interactive demonstration
+  
+- **build/**: Contains the build process for this repository, implemented using FAKE and EasyBuild.
+
+## Configuration Files
+
+This project uses MSBuild's Directory.Build.props system for configuration:
+
+### Directory.UserConfig.props
+
+This is where you place configuration unique to your project:
 
-## Configuration files
```

Comment:
Based on my comments below, I think we can revert this section to the previous state.